src/rpc_server.rs

```diff
@@ -196,6 +197,9 @@ pub trait RPC {
 
     /// Gracious shutdown.
     async fn shutdown() -> bool;
+
+    /// Get CPU temperature.
+    async fn get_cpu_temps() -> Option<f64>;
```

Comment:
I suggested naming it get_cpu_temps (plural) because I was expecting there would be a Vec of temps, one for each CPU core.  I see now that systemstat only returns a single value which surprises me.   But in light of that, perhaps its best just to name it `cpu_temp`.  I think the "get_" is superfluous.

Reply:
You are right @dan-da .